ios/PurchasesHybridCommon/PurchasesHybridCommonUI/PaywallProxy.swift

```diff
@@ -183,6 +183,10 @@ import UIKit
 
 @available(iOS 15.0, *)
 extension PaywallProxy: PaywallViewControllerDelegate {
+    
+    public func paywallViewControllerDidStartPurchase(_ controller: PaywallViewController) {
```

Comment:
Do we also need one for restore starting?

Reply:
yes, but that one is not available in purchases-ios yet so we need to add it there first 😢